### ISSUE TITLE
[APP SUBMITTED]: 'ascii' codec can't encode character u'\u2013' in position 80: ordinal not in range(128)

### ISSUE THREAD

**Issue by FelipeSQ** [SiCKRAGETV/sickrage-issues#1162]
### INFO
Python Version: **2.7.3 (default, Apr 12 2012, 13:08:47) [GCC 4.2.3]**
Operating System: **Linux-3.4.6-armv5tel-with-glibc2.4**
Locale: unknown
Branch: **master**
Commit: SiCKRAGETV/SickRage@669046ea40a7a26a6121a1b07721f3c1c3cf5b10
Link to Log: https://gist.github.com/ea17500770bb9dea0961
### ERROR
```
POSTPROCESSER :: Exception generated in thread POSTPROCESSER: ascii : /share/MD0_DATA/Multimedia/TV Shows/Better Call Saul/Season 01/Better Call Saul – S01E10 – Marco.mp4 : error 80 : error 81 : ordinal not in range(128)
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by abeloin** [comment]
@FelipeSQ What is your system locale?

**Comment by FelipeSQ** [comment]
To be honest, I had followed up the instructions by echel0n here: http://tinyurl.com/n634bvw. But I just realized that the changes I did to my .bashrc and to my .profile on my QNAP were lost. So, since yesterday I created a new autorun script that is called on startup, I just added this there and will see if I still have those errors. But I forgot to do an ECHO on them before running the script to test, so that means that I don't know what was the locale before that. I think we should just close this and if this still happens, I will get those values before submitting the error. Thank you very much!
